### ISSUE TITLE
tko-01-beyond-cka/lessons/lab-02-deploy-applications-using-kubernetes - gowebapp-mysql doesn't start up due to missing env variable 

### ISSUE THREAD

**Issue by rabeyta** [vmware-tanzu-learning/tko-ref-arch-training#26]
**Describe the issue**
gowebapp-mysql doesn't start up using the spec due to missing env variable

**To reproduce**
1. create gowebapp-mysql-deployment.yaml as spec'd

**Expected behavior**
the app to startup

**Screenshots**
```
[~/gowebapp/gowebapp-mysql] $ kubectl get deployment -l "app=gowebapp-mysql"
NAME             READY   UP-TO-DATE   AVAILABLE   AGE
gowebapp-mysql   0/1     1            0           24s
[~/gowebapp/gowebapp-mysql] $ k get po
NAME                              READY   STATUS             RESTARTS   AGE
gowebapp-mysql-6ccffccbc6-5xqjz   0/1     CrashLoopBackOff   1          16s
[~/gowebapp/gowebapp-mysql] $ k logs gowebapp-mysql-6ccffccbc6-5xqjz 
2022-04-08 18:49:15+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 5.6.51-1debian9 started.
2022-04-08 18:49:15+00:00 [ERROR] [Entrypoint]: Database is uninitialized and password option is not specified
    You need to specify one of the following:
    - MYSQL_ROOT_PASSWORD
    - MYSQL_ALLOW_EMPTY_PASSWORD
    - MYSQL_RANDOM_ROOT_PASSWORD
[~/gowebapp/gowebapp-mysql] $ 
```

**Additional context**
adding env variable of 
```
- name: MYSQL_ROOT_PASSWORD
  value: mypassword
```
solved the issue for me

**Comment by djschny** [comment]
Thanks for taking the time to submit this. 

Yes indeed, I can trace that back to the commit/PR with the issue. Working on having the fix udpated.

**Comment by djschny** [comment]
This was fixed in `v1.6.4-beta2` of the Foundations course. That was then deployed to KubeAcademy staging.
